### DISCUSSION TITLE
Make autocomplete with tui-input and tui-data-list show list on focus/click

### DISCUSSION THREAD

**Discussion by m-aditya** [taiga-family/taiga-ui#624] [Q&A] [answered]
I've been trying to figure this out for a while. I have an autocomplete that I would like the initial set of options to display on focus/click as apposed to having to type a character first to see the list. Any help would be greatly appreciated!

**Comment by waterplea** [comment, marked as answer]
Input + DataList is autocomplete. This, as the name states, completes your input automatically. Therefor the suggestions do not pop up instantly, only when you type. You can always open them by pressing down key. If you, instead, want to pick an item and have a way to filter the options by text field, perhaps you are looking for a [ComboBox](https://taiga-ui.dev/components/combo-box)?

**Reply by m-aditya** [reply]
I was indeed looking for a combobox. Thank you very much!
